<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anaPencere</class>
 <widget class="QWidget" name="anaPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TAŞ KAĞIT MAKAS</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="sayac">
          <property name="text">
           <string>SAYAÇ = 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioMakas">
          <property name="text">
           <string>Makas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTas">
          <property name="text">
           <string>Taş</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioKagit">
          <property name="text">
           <string>Kağıt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>PC SEÇİMİ :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>DURUM : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>SİZ:                     BİLGİSAYAR:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
